src/ResourceManager/PowerBI/AzureRM.PowerBI.psd1

```diff
@@ -0,0 +1,132 @@
+#
```

Comment:
@selasagi can this file be remove? I thought the cmdlets were being shipped with `AzureRM.PowerBIEmbedded`

Reply:
removed